File: julia/sample1.jl
    using Einsum

A = zeros(5, 6, 7)
X = randn(5, 2)
Y = randn(6, 2)
Z = randn(7, 2)
@einsum A[i, j, k] = X[i, r] * Y[j, r] * Z[k, r]

println(size(A))

File: python/sample1.py
# -*- coding: utf-8 -*-

import numpy as np

a = np.random.normal(size=5*2).reshape((5, 2))
b = np.random.normal(size=6*2).reshape((6, 2))
c = np.random.normal(size=7*2).reshape((7, 2))
ABC = np.einsum("ij,kj,lj->ikl", a, b, c)
print(ABC.shape)
